.babelrc

```diff
@@ -14,6 +14,7 @@
   "env": {
     "test": {
       "plugins": [
+        "@babel/transform-runtime",
```

Comment:
This plugin is added because the newly added **async** `jest` test was failing due to its absence.
As tests are for developers, hence added this dependency as a devDependency.